newdle/newdle.cfg.example

```diff
@@ -89,3 +89,9 @@ EXCHANGE_DOMAIN = ''
 EXCHANGE_PROVIDER_SERVER = ''
 EXCHANGE_PROVIDER_ACCOUNT = ''
 EXCHANGE_PROVIDER_CREDENTIALS = ('', '')
+
+# Cleanup configuration (values are in days)
+# Days after which an incomplete newdle is cleaned up after its last update.
+LAST_ACTIVITY_CLEANUP_INTERVAL = 90
```

Comment:
Should we allow (and maybe even default to) `None` to disable the cleanup?

That would also allow people to set only one of the two settings to clean up based on final date or last activity *only*.

Reply:
I changed the logic a bit around, please take a look.